rpc_util.go

```diff
@@ -502,4 +502,10 @@ const (
 // Version is the current grpc version.
 const Version = "1.9.0-dev"
 
+// These constants specify the minimum and maximum acceptable RPC version values.
```

Comment:
I believe this comment is fine if you do:
```go
const (
  MaxRPCVersion...
  ...
)
```

Otherwise, this must be formatted like `// MaxRPCVersionMajor contains the maximum .............` for golint.